boot/bootutil/include/bootutil/bootutil_public.h

```diff
@@ -71,7 +71,17 @@ extern "C" {
 /** Swapping encountered an unrecoverable error */
 #define BOOT_SWAP_TYPE_PANIC    0xff
 
+#define BOOT_MAGIC_SZ           16
```

Comment:
Remove `#define BOOT_MAGIC_SZ (sizeof boot_img_magic)`? (at line 90)

Reply:
```suggestion
#define BOOT_MAGIC_SZ           (sizeof boot_img_magic)
```
Actually the most correct would be to bring line 90 upwards.

Reply:
`BOOT_MAGIC_SZ` needs to be resolved at preprocessing time due to a new change introduced in this PR and `sizeof` is only resolved at compile time.
So I had to leave it as **16**